### PR TITLE
Add PRTS submission

### PR DESCRIPTION
**Pull request by breez3young**
## Submission checklist

- [x] I added exactly one JSON file for this submission
- [x] I placed the file in the `submissions/` directory
- [x] My JSON follows the submission template in the README
- [x] I used `"N/A"` for any field that is unavailable or not applicable
- [x] I did not modify existing submission files unrelated to this PR

## Notes (optional)

Add any extra context for reviewers here. **Submission details are auto-generated from your JSON** in a bot comment on this PR (updated when you push changes).

### CODE REVIEW

**Comment by sepnasiriany** [comment]
thanks for submitting @breez3young! The PRTS repo doesn't seem to have robocasa365 eval code/script, could you please share how you evaluated your policy on robocasa365?

**Comment by breez3young** [comment]
Hi @sepnasiriany ,

Sorry for the missing part — I've just pushed the RoboCasa-365 evaluation code and configs to the `eval/robocasa` branch:

https://github.com/TeleHuman/PRTS/tree/eval/robocasa/examples/robocasa365

These are the re-organized scripts we used to produce the reported numbers.

**Comment by sepnasiriany** [comment]
I finished the evals @breez3young, the only number that was a little off was composite_seen, would you like me to use these results?

```
[atomic_seen]
CloseBlenderLid               33.33%
CloseFridge                   96.67%
CloseToasterOvenDoor          46.67%
CoffeeSetupMug                46.67%
NavigateKitchen               30.00%
OpenCabinet                   80.00%
OpenDrawer                    63.33%
OpenStandMixerHead            86.67%
PickPlaceCounterToCabinet     60.00%
PickPlaceCounterToStove       66.67%
PickPlaceDrawerToCounter      80.00%
PickPlaceSinkToCounter        96.67%
PickPlaceToasterToCounter     90.00%
SlideDishwasherRack           83.33%
TurnOffStove                  36.67%
TurnOnElectricKettle          70.00%
TurnOnMicrowave               60.00%
TurnOnSinkFaucet              66.67%
AGGREGATE                     66.30% (358/540)

[composite_seen]
DeliverStraw                   6.67%
GetToastedBread                6.67%
KettleBoiling                 23.33%
LoadDishwasher                60.00%
PackIdenticalLunches          33.33%
PreSoakPan                    43.33%
PrepareCoffee                  0.00%
RinseSinkBasin                23.33%
ScrubCuttingBoard             40.00%
SearingMeat                    0.00%
SetUpCuttingStation           56.67%
StackBowlsCabinet             53.33%
SteamInMicrowave              10.00%
StirVegetables                20.00%
StoreLeftoversInBowl          26.67%
WashLettuce                   23.33%
AGGREGATE                     26.67% (128/480)

[composite_unseen]
ArrangeBreadBasket            40.00%
ArrangeTea                    36.67%
BreadSelection                16.67%
CategorizeCondiments           3.33%
CuttingToolSelection          53.33%
GarnishPancake                16.67%
GatherTableware                0.00%
HeatKebabSandwich              0.00%
MakeIceLemonade               20.00%
PanTransfer                    0.00%
PortionHotDogs                13.33%
RecycleBottlesByType          13.33%
SeparateFreezerRack           16.67%
WaffleReheat                  13.33%
WashFruitColander             43.33%
WeighIngredients              13.33%
AGGREGATE                     18.75% (90/480)
```

**Comment by breez3young** [comment]
Hi, @sepnasiriany ,

I noticed that the composite_seen score differs from our reported result. From the aggregate counts, it looks like the current evaluation used 30 episodes per task, whereas our reported results were obtained with 50 episodes per task (to robustly produce our results, we deliberately set seeds 0–49).

The reason why we test 50 episodes per task is that we follow the standard protocol in [official document](https://robocasa.ai/docs/build/html/benchmarking/benchmarking_overview.html#:~:text=For%20each%20benchmarking%20experiment%20we%20specify%20the%20specific%20task,report%20average%20task%20success%20rates%20across%20these%2050%20rollouts) to sample 50 scenarios to run evaluation rollouts on and report average task success rates across these 50 rollouts in each task.

For a like-for-like comparison, could you try to use episodes_per_task=50, and the same RoboCasa/robosuite/harness versions and success-check settings?

**Comment by breez3young** [comment]
For reference, here are the per-task evaluation results underlying our submission. Each task was evaluated for 50 episodes using seeds 0–49.
```text
[atomic_seen]
CloseBlenderLid               36.00%
CloseFridge                   94.00%
CloseToasterOvenDoor          58.00%
CoffeeSetupMug                46.00%
NavigateKitchen               36.00%
OpenCabinet                   80.00%
OpenDrawer                    58.00%
OpenStandMixerHead            80.00%
PickPlaceCounterToCabinet     64.00%
PickPlaceCounterToStove       68.00%
PickPlaceDrawerToCounter      82.00%
PickPlaceSinkToCounter        88.00%
PickPlaceToasterToCounter     90.00%
SlideDishwasherRack           72.00%
TurnOffStove                  28.00%
TurnOnElectricKettle          84.00%
TurnOnMicrowave               42.00%
TurnOnSinkFaucet              76.00%
AGGREGATE                     65.67% (591/900)

[composite_seen]
DeliverStraw                  18.00%
GetToastedBread                2.00%
KettleBoiling                 34.00%
LoadDishwasher                50.00%
PackIdenticalLunches          26.00%
PreSoakPan                    66.00%
PrepareCoffee                  4.00%
RinseSinkBasin                30.00%
ScrubCuttingBoard             50.00%
SearingMeat                    0.00%
SetUpCuttingStation            64.00%
StackBowlsCabinet             62.00%
SteamInMicrowave              18.00%
StirVegetables                30.00%
StoreLeftoversInBowl          40.00%
WashLettuce                   40.00%
AGGREGATE                     33.38% (267/800)

[composite_unseen]
ArrangeBreadBasket             36.00%
ArrangeTea                     28.00%
BreadSelection                 22.00%
CategorizeCondiments            6.00%
CuttingToolSelection           38.00%
GarnishPancake                 12.00%
GatherTableware                 0.00%
HeatKebabSandwich               0.00%
MakeIceLemonade                46.00%
PanTransfer                     0.00%
PortionHotDogs                 12.00%
RecycleBottlesByType           14.00%
SeparateFreezerRack            16.00%
WaffleReheat                   16.00%
WashFruitColander              46.00%
WeighIngredients               18.00%
AGGREGATE                     19.38% (155/800)
```

**Comment by sepnasiriany** [comment]
You are right to point out the 50 episodes in our documentation, but this was used for the baseline policies published in our paper and in the interest of saving time and compute, I consistently evaluated 30 episodes per task across all leaderboard externally published models. In any case, I will run an additional 20 evals per task for `composite_seen` to see if results meaningfully change. 

On another note, please provide a detailed description of your training setup (if you only trained on `pretrain_human300`, denoising steps, other hyperparameters, etc)

**Comment by breez3young** [comment]
Thank you for the clarification. I appreciate you running the additional 20 episodes for `composite_seen`.

Regarding our training setup, we followed the [Benchmark instructions in the RoboCasa multi-task learning documentation](https://robocasa.ai/docs/build/html/benchmarking/multitask_learning.html).

Here is our training setup on RoboCasa365:

- **Training data:** `pretrain_human300` only. We did not use target-split data.
- **Initialization:** initialized from the [PRTS-4B checkpoint](https://huggingface.co/TeleEmbodied/PRTS-4B) and fine-tuned to obtain PRTS-RoboCasa365.
- **Action chunk size:** 32.
- **State normalization:** `MIN_MAX`.
- **Action normalization:** `MEAN_STD`.
- **Diffusion denoising steps:** 5.
- **Batch size:** 12 samples per device, 16 H100 GPUs, totalling a global batch size of 192.
- **Learning rates:**
  - Action expert: `1e-4`
  - VLM backbone: `1e-5`
- **Training schedule:** the run was configured for 300,000 optimizer steps.
- **Checkpoint used for the reported results:** the checkpoint at 250,000 optimizer steps.
- **Evaluation split:** `pretrain`, covering the `atomic_seen`, `composite_seen`, and `composite_unseen` task sets.

Although the training job was configured to run for 300k steps, the results in our submission were obtained using the 250k-step checkpoint.

**Comment by sepnasiriany** [comment]
here are the updated `composite_seen` results with 50 rollouts per task:

```
[composite_seen] 
DeliverStraw                  12.00%
GetToastedBread                6.00%
KettleBoiling                 26.00%
LoadDishwasher                54.00%
PackIdenticalLunches          26.00%
PreSoakPan                    48.00%
PrepareCoffee                  2.00%                       
RinseSinkBasin                46.00%
ScrubCuttingBoard             44.00%
SearingMeat                    0.00%
SetUpCuttingStation           52.00%
StackBowlsCabinet             58.00%
SteamInMicrowave              16.00%
StirVegetables                28.00%                       
StoreLeftoversInBowl          40.00%
WashLettuce                   26.00%
AGGREGATE                     30.25% (242/800)
```

This still falls a little bit short of the 33% given above, but definitely better than before
